integrations/ollama/src/haystack_integrations/components/embedders/ollama/text_embedder.py

```diff
@@ -66,3 +66,25 @@ def run(self, text: str, generation_kwargs: Optional[Dict[str, Any]] = None):
         result["meta"] = {"model": self.model}
 
         return result
+
+    @component.output_types(embedding=List[float], meta=Dict[str, Any])
+    async def run_async(self, text: str, generation_kwargs: Optional[Dict[str, Any]] = None):
+        """
+        Asynchronously run an Ollama Model to compute embeddings of the provided text.
+
+        :param text:
+            Text to be converted to an embedding.
+        :param generation_kwargs:
+            Optional arguments to pass to the Ollama generation endpoint, such as temperature,
+            top_p, etc. See the
+            [Ollama docs](https://github.com/jmorganca/ollama/blob/main/docs/modelfile.md#valid-parameters-and-values).
+        :returns: A dictionary with the following keys:
+            - `embedding`: The computed embeddings
+            - `meta`: The metadata collected during the embedding process
+        """
+        self._client = AsyncClient(host=self.url, timeout=self.timeout)
```

Comment:
This will create a new `AsyncClient` instance each time `run_async` is called, which can be suboptimal. 

What do you think about initialising an async client instance in `__init__` instead and use it directly in `run_async`?